packages/x-date-pickers/src/MobileDatePicker/tests/MobileDatePicker.test.tsx

```diff
@@ -113,54 +112,7 @@ describe('<MobileDatePicker />', () => {
     });
   });
 
-  describe('Slots: Toolbar', () => {
-    it('should render custom toolbar component', () => {
-      render(
-        <MobileDatePicker
-          open
-          slots={{
-            toolbar: () => <div data-testid="custom-toolbar" />,
-          }}
-        />,
-      );
-
-      expect(screen.getByTestId('custom-toolbar')).toBeVisible();
-    });
-
-    it('should format toolbar according to `toolbarFormat` prop', () => {
-      render(
-        <MobileDatePicker
-          open
-          defaultValue={adapterToUse.date(new Date(2018, 0, 1))}
-          slotProps={{
-            toolbar: {
-              toolbarFormat: 'MMMM',
-            },
-          }}
-        />,
-      );
-
-      expect(screen.getByMuiTest('datepicker-toolbar-date').textContent).to.equal('January');
-    });
-  });
-
-  describe('Component slots: Day', () => {
-    it('should render custom day', () => {
-      render(
-        <MobileDatePicker
-          open
-          defaultValue={adapterToUse.date(new Date(2018, 0, 1))}
-          slots={{
-            day: (props) => <PickersDay {...props} data-testid="test-day" />,
-          }}
```

Comment:
Since this test is exactly the same as the one below, I'm wondering if i's not a mistake in this test should use `components` instead of `slots`

Reply:
For me we add tests duplication due to a bad git merge at some point, this PR removes those duplications :+1: